### ISSUE TITLE
Carbon Exception 5.6.10

### ISSUE THREAD

**Issue by ollywarren** [laravel/framework#23465]
- Laravel Version: 5.6.10
- PHP Version: 7.2
- Database Driver & Version: MySQL 5.7

### Description:
Seems the latest update to 5.6.10 and Carbon 1.24.1 might have a bug got this after running a composer update whilst doing some package development:

```

Loading composer repositories with package information
Updating dependencies (including require-dev)
Package operations: 0 installs, 2 updates, 0 removals
  - Updating nesbot/carbon (1.23.0 => 1.24.1): Downloading (100%)
  - Updating laravel/framework (v5.6.9 => v5.6.10): Downloading (100%)
Writing lock file
Generating optimized autoload files
> Illuminate\Foundation\ComposerScripts::postAutoloadDump
> @php artisan package:discover

   ErrorException  : Declaration of Illuminate\Support\Carbon::__set_state(array $array) should be compatible with Carbon\Carbon::__set_state($state)

```

If i lock to 5.6.9 the issue goes away.

### Steps To Reproduce:

```composer update```

**Comment by 36864** [comment]
This is being handled.
https://github.com/laravel/framework/commit/67d8a4b15ffdeeacc2c27efad05735a59dba1c44
https://github.com/laravel/framework/pull/23464

**Comment by thisprojectworks** [comment]
Exactly same issue here with same enviorment.

**Comment by ollywarren** [comment]
Thanks guys, just seen the PR.

https://github.com/laravel/framework/pull/23464

**Comment by anare** [comment]
https://github.com/briannesbitt/Carbon/commit/ef12cbd8ba5ce624f0a6a95e0850c4cc13e71e41

**Comment by brunogoncalves** [comment]
Some solution?
I have the same problem.

**Comment by dash8x** [comment]
Getting the same error

**Comment by thisprojectworks** [comment]
For a momentary solution, you can rollback to Laravel 5.6.9 and it works.

**Comment by jozeflambrecht** [comment]
same for LTS 5.5.37 ...

**Comment by jpriceonline** [comment]
Getting the same error with exactly the same environment. Though, setting to 5.6.9 seems to have done the trick.

**Comment by them-roc** [comment]
just rolled back from 5.5.37 to 5.5.36

**Comment by RobinBertilsson** [comment]
Yup, had to do the same, rolled back to 5.5.36 just like @them-roc said.

**Comment by GrahamCampbell** [comment]
Fixed in 5.5.39 and 5.6.11.
